AdventOfCode.Runner/Input/README.MD

```diff
@@ -0,0 +1 @@
+﻿Drop your input files here with the name `Day1.txt`, `Day2.txt`, etc.
```

Comment:
Interesting does `.txt` file just need to be some kind of text file with strings?

Reply:
AoC usually provides the user-specific input as text. That's why I've added a .gitignore entry for these, so nobody accidentally checks it in.

Reply:
If you look at my `Day<>` design you can see I specify a function to convert string into a more-appropriate input format. This keeps the code cleaner when you want to do stupid shit like solve these problems in a single-line linq statement.

Reply:
haha i see nice work!